testsuite/features/step_definitions/command_steps.rb

```diff
@@ -580,12 +580,17 @@
     raise ScriptError, 'File injection failed' unless success
   end
   server.run('chmod +x /etc/ipmi/fake_ipmi_host.sh', verbose: true, check_errors: true)
-  server.run('ipmi_sim -n < /dev/null > /dev/null &', verbose: true, check_errors: true)
+  # Check if ipmi_sim is already running
+  if server.run('pgrep -f ipmi_sim', verbose: false, check_errors: false).empty?
```

Comment:
I think this `.empty?` like will not work, as we return more than one value on run() method.
It might work with `run(xx)[0].empty?`